site/content/docs/traces-open-telemetry/importing-traces/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Send your traces to Checkly
+weight: 20
+menu:
+  platform:
+    parent: "Traces (beta)"
+beta: true
+---
+
+Send your 3rd party backend OpenTelemetry traces to Checkly. Inspect and drill down to why a check failed or degraded.
+<!--more-->
+
+In [the Traces tab your Checkly account settings](https://app.checklyhq.com/settings/account/traces): 
```

Comment:
Should this be something like 'In the Traces tab, in your Checkly account settings'?

Reply:
```suggestion
In [the Traces tab, in your Checkly account settings](https://app.checklyhq.com/settings/account/traces): 
```

Reply:
Totally, great catch 💯